config/crd/bases/awx.ansible.com_awxs.yaml

```
@@ -1811,6 +1789,12 @@ spec:
                 type: array
                 items:
                   type: string
+              postgres_data_volume_init:
+                description: Sets permissions on the /var/lib/postgresql/data for postgres container using an init container (not Openshift)
```

Is it `/var/lib/postgresql/data` or `/var/lib/pgsql/data` ?
```suggestion
                description: Sets permissions on the /var/lib/pgdata/data for postgres container using an init container (not Openshift)
```